features/admin.users.v1/api/users.ts

```diff
@@ -470,3 +471,46 @@ export const terminateAllUserSessions = (userId: string): Promise<AxiosResponse>
                 error.config);
         });
 };
+
+/**
+ * Resends the verification code for the for recovery-related scenarios.
+ *
+ * @param data - The request payload containing user information and properties.
+ * @returns A promise containing the response.
+ * @throws `IdentityAppsApiException` if the request fails or if the response status is not as expected.
+ */
+export const resendCode = (data: ResendCodeRequest): Promise<any> => {
```

Comment:
Promise<any> create a response interface and pass

Reply:
Move to admin.core.v1

Reply:
resendCode is specific to user management, isn't it? any specific reason to move this to core.v1?

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/399e7573e645d5c76ea0e9c6b3a47780a2c40019 . Further had an offline discussion with Jayashakthi and agreed to keep resendCode api in `users.v1`